fineract-provider/src/main/java/org/apache/fineract/portfolio/client/service/ClientWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -430,7 +430,8 @@ public CommandProcessingResult updateClient(final Long clientId, final JsonComma
             }
 
             final ExternalId externalId = externalIdFactory.createFromCommand(command, ClientApiConstants.externalIdParamName);
-            if (command.isChangeInStringParameterNamed(ClientApiConstants.externalIdParamName, externalId.getValue())) {
+            if (command.isChangeInStringParameterNamed(ClientApiConstants.externalIdParamName,
```

Comment:
Please raise a separate PR for fixing the client external id updating bug!

Reply:
Done